kubemarine/plugins/yaml/calico-v3.24.2.yaml.original

```diff
@@ -963,10 +963,10 @@ spec:
                 - type: string
                 description: 'BPFPSNATPorts sets the range from which we randomly
                   pick a port if there is a source port collision. This should be
-                  within the ephemeral range as defined by RFC 6056 (1024–65535) and
+                  within the ephemeral range as defined by RFC 6056 (1024-65535) and
                   preferably outside the  ephemeral ranges used by common operating
-                  systems. Linux uses 32768–60999, while others mostly use the IANA
-                  defined range 49152–65535. It is not necessarily a problem if this
+                  systems. Linux uses 32768-60999, while others mostly use the IANA
+                  defined range 49152-65535. It is not necessarily a problem if this
```

Comment:
This is to resolve issue https://github.com/paramiko/paramiko/issues/1133
So this makes original calico.yaml not exactly "original".
@alexarefev please pay note on this.

Reply:
@ilia1243 pls do the following:
- update https://git.netcracker.com/PROD.Platform.HA/KubeMarine/-/blob/master/documentation/public/Update-thirdparties.md#calico  section with step how to update newly adopted version
- take  #342 issue  with priority and plan it in next iteration

Reply:
Done